### ISSUE TITLE
ActiveX error occurs intermittently when using Create Instance action

### ISSUE THREAD

**Issue by jjentges** [blue-prism/ms-excel-utility#12]
When running the 'Create Instance' action from this VBO, the action sometimes fails with the following error: "Could not execute code stage because exception thrown by code stage: Cannot create ActiveX component."

This error does not happen every time the action runs, but it causes processes to fail when the error does occur. I would like to understand what solutions are available to make sure this error does not occur at all.

**Comment by shashankkumar88** [comment]
@jjentges Can you try reinstalling MS Office on the machine on which you are getting this issue.

**Comment by jjentges** [comment]
I have tried uninstalling and reinstalling Office, but the errors still come up. The machine is currently running Office Version 2008, Build 13127.21624.

**Comment by EricWilson-BluePrism** [comment]
To clarify, this is actually Microsoft 365, correct? "Office Version 2008" is a bit misleading as that was actually the name for Microsoft's Office for Mac release back in 2008.

**Comment by jjentges** [comment]
You are correct, I am running Microsoft 365. When I open an Office app and go to File > Account, the version information is v2008, build 13127.21624
